### ISSUE TITLE
ABC iView error

### ISSUE THREAD

**Issue by jpdehollain** [xbmc-catchuptv-au/repo#44]
I tried playing a couple of links from the ABC iView add-on and kept getting an error. I've uploaded the full log file, but I believe the relevant errors I get are:
ERROR: CCurlFile::Stat - Failed: HTTP response code said error(22) for https://abcsecurehttp...
ERROR: CCurlFile::FillBuffer - Failed: HTTP returned error 403
ERROR: CCurlFile::Open failed with code 403 for https://abcsecurehttp...

I live in the Netherlands, so it could be related to geo-blocking, but it's odd because I've been using this add-on without problems recently.
[kodi.log.txt](https://github.com/xbmc-catchuptv-au/repo/files/587216/kodi.log.txt)

**Comment by andybotting** [comment]
I was under the impression that everything was geoblocked. You should look into a vpn or smart DNS service to get you access

**Comment by NikolaiHampton** [comment]
I may be able to confirm this (from within Australia). A clean Kodi 17.1 as of this morning, no other plugins. 

I added ABC iView and SBS on Demand via the catchup tv repo.

SBS On Demand works fine.

ABC iView browses ok, but attempting to play results in nothing played.

kodi.log from plugin launch to "Playback Failed" error:

https://pastebin.com/raw/15e8znWg

**Comment by NikolaiHampton** [comment]
This appears to be transient. It started working with no changes.

**Comment by andybotting** [comment]
@NikolaiHampton yeah, I think their API is struggling with some requests. I've got some new code I'm working on with some clever retry mechanisms, so hopefully this will be less of an issue in the future.
